src/Requests.jl

```diff
@@ -153,6 +156,11 @@
         r.finished = true
         close(p.sock)
 
+        #If MIME type allows, convert r.data to a Julia string
+        mimetype = r.headers["Content-Type"]
+        if (contains(mimetype, "charset=utf-8") || contains(mimetype, "charset=us-ascii"))
```

Comment:
It is better to look for `text/` substring rather then `charset`, e.g. `text/html` or `text/plain`.